### DISCUSSION TITLE
Pending for a while "Needs More Details"

### DISCUSSION THREAD

**Discussion by STOL4S** [community/community#60545] [New to GitHub] [answered]
I filled everything out that was asked but my account still says pending after a week or 2 and says they require additional info but no email was received and when expanding the window it says ive completed all tasks and my application is pending. Is this normal?

**Comment by OshekharO** [comment, marked as answer]
Contact the GitHub customer support or the appropriate department responsible for account verification. Provide them with the details of your situation and inquire about the status of your account application. They should be able to assist you further and provide clarification on any additional steps or information required.

**Reply by STOL4S** [reply]
Thank you +1
